app/konnect/dev-portal/access-and-approval/azure.md

```diff
@@ -53,6 +53,8 @@ Group claims automatically add or remove users from group memberships. To config
 
 1. Click **Add**.
 
+<!-- I'm not sure this still needs to be here? if yes, need to rewrite so that it only covers what's needed specifically for DevPortal
```

Comment:
@iamnande Do you know if the content that was commented out here still needs to be included in this page?

Reply:
@iamnande Do you know if the content that was commented out here still needs to be included in this page?